packages/core/src/app/order/OrderSummaryPrice.spec.tsx

```diff
@@ -1,23 +1,51 @@
-import { shallow, ShallowWrapper } from 'enzyme';
+import { CheckoutSelectors, CheckoutService, createCheckoutService } from '@bigcommerce/checkout-sdk';
+import { mount } from 'enzyme';
 import React from 'react';
 
+import { CheckoutProvider } from '@bigcommerce/checkout/payment-integration-api';
+
 import { ShopperCurrency } from '../currency';
 
-import OrderSummaryPrice from './OrderSummaryPrice';
+jest.mock('react-transition-group', () => ({
+    CSSTransition: ({ children }: { children: React.ReactNode }) => children,
+}));
+
+import OrderSummaryPrice, { OrderSummaryPriceProps, WithCheckoutOrderSummaryPriceProps } from './OrderSummaryPrice';
 
 describe('OrderSummaryPrice', () => {
-    let orderSummaryPrice: ShallowWrapper;
+    let orderSummaryPrice: any;
+    let checkoutService: CheckoutService;
+    let checkoutState: CheckoutSelectors;
+
+    const mountTestComponent = (
+        props: OrderSummaryPriceProps & WithCheckoutOrderSummaryPriceProps & { children?: React.ReactNode }
+    ) => {
+        const orderSummaryPriceContainer = mount(
+            <CheckoutProvider checkoutService={checkoutService}>
```

Comment:
Another one.

Reply:
the same as before, it is a test with enzyme